### ISSUE TITLE
Set form action

### ISSUE THREAD

**Issue by teckaa** [FlutterFlow/flutterflow-issues#3748]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Before the last update in FlutterFlow, my project was working fine. However, after receiving a message to update and refresh, this issue started appearing. Even when I retry to set the form action, I keep getting an error saying, "Invalid form field selected. Please select a different field."

<img width="418" alt="Screenshot 2024-09-04 at 10 09 56 PM" src="https://github.com/user-attachments/assets/5a5a754b-f487-47ff-b7e1-aa33fa7c0f86">
<img width="799" alt="Screenshot 2024-09-04 at 10 13 22 PM" src="https://github.com/user-attachments/assets/8f06b2a9-287c-4711-a3e5-7eec82986cf4">
<img width="939" alt="Screenshot 2024-09-04 at 10 23 38 PM" src="https://github.com/user-attachments/assets/7ea49570-e405-41fe-93e9-2e24c2ae96cc">

### Expected Behavior

To work just as normal

### Steps to Reproduce

Add a textfield
Add button and action to the button
Set field to pass data the to field
After setting or passing the data to the field, this error keep coming up "Invalid form field selected. Please select a different field."

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESlsrlx4xgoswA+aXcLcZQhQIgQ0QnU7gnlO1ucx4jG5CuO4l/PM/7bUNKQvWmfVA7IlmKkXEe+evGudnxD/dcPkysGaZA0JVAWxXyYFyRVJi1OqWzPX98GfNQGX6M14mvqxJ5APpaWEIH1ziXNq3qNleeY8aSfxBlZ7vfcPo=

### Visual documentation

I share a video from a blank project and the other one from my old project, the blank project work fine. but in my project, am getting error.

https://github.com/user-attachments/assets/37ac862b-3e58-4d94-b042-c0c6f52c3dbd

https://github.com/user-attachments/assets/7245b031-5071-477e-aa87-983e65c860d6

### Environment

```markdown
- FlutterFlow version: 
- Platform:
- Browser name and version:
- Operating system and version affected:
```

**Comment by ALaiFF** [comment]
The same issue was encountered right after the most recent update...

**Comment by rzambroni** [comment]
Hi everyone,

Thank you for bringing this issue to our attention. I’ve passed it along to our engineering team, and they’ll be looking into it as soon as possible. We’ll make sure to keep you updated with any progress or next steps here.

**Comment by hamreen** [comment]
Before the update, everything was working as expected. After the update, my app now throws the "Invalid form field selected" error on multiple pages, making the project non-functional in its current state. Rebinding the form fields doesn't resolve the issue.

This bug is quite critical for my project since we're on a tight timeline.

**Comment by MahamadKanabi** [comment]
We're really dependent on this functionality, and the issue is preventing us from moving forward. It's quite surprising that such a significant problem was introduced, especially considering how vital form actions are to most apps.

**Comment by MahamadKanabi** [comment]
We're really dependent on this functionality, and the issue is preventing us from moving forward. It's quite surprising that such a significant problem was introduced, especially considering how vital form actions are to most apps.

**Comment by labraxoid** [comment]
I confirm the bug, the settings of all Set form actions in the entire project have fallen off, reconfiguration does not help, we cannot move on

**Comment by Mdmuzammil16** [comment]
Got the same issue unable to do anything because of this issue can your team please give a timeline when it will be fixed??

**Comment by htsnet** [comment]
Same issue here. The app was OK yesterday and today I received 129 errors of form field action.

**Comment by flutterdevcodiste** [comment]
The same issue occurred right after updating to the latest FlutterFlow version. With 1.5 million users, it's crucial to ensure every release is stable. A single issue from your end is causing delays for many, including my work.

**Comment by nev-21** [comment]
Yes, this is critical. We can't run any app.

**Comment by yashcx** [comment]
We're encountering same issues. It seems that FlutterFlow is not recognizing any new updates to the builds, and no changes are being pushed from the desktop app (in my case, a Mac). We heavily rely on the Desktop app.

**Comment by rzambroni** [comment]
Hi there,

Just wanted to let you know that a fix is on the way. We’re sorry for the trouble and really appreciate your patience while we get this sorted.

**Comment by labraxoid** [comment]
the bug was fixed in version 4.1.87

**Comment by ALaiFF** [comment]
FlutterFlow v4.1.86+ (yesterday's update). The bug seems to have been fixed. There are no more errors
